extensions/cloud/portability-cloud-google/src/main/java/org/datatransferproject/cloud/google/GoogleJobStore.java

```diff
@@ -327,10 +300,62 @@ private Key getDataKey(UUID jobId, String key) {
     if (entity == null) {
       return null;
```

Comment:
I thought we were moving all generic-type storage to the new UserDataStore (via pass thru in the existing JobStore interface), while PortabilityJob stuff (i.e. job metadata) would continue to be stored in DataStore (the dataStore variable). Is all the "type T" generic stuff == job temp data? +@holachuy